server/pairing/client.go

```diff
@@ -496,6 +496,12 @@ func setupReceivingClient(backend *api.GethStatusBackend, cs, configJSON string)
 		return nil, err
 	}
 
+	// ignore err because we don't care if there is no active account
+	activeAccount, _ := backend.GetActiveAccount()
+	if activeAccount != nil {
+		conf.ReceiverConfig.LoggedInKeyUID = activeAccount.KeyUID
```

Comment:
@qfrank as far as I know the only way to create an account on another device, by syncing from the device where that account is already created, is possible only if there is no info about that account on another device at that moment. That means `backend.GetActiveAccount()` will always be `nil`. `KeyUID` on the receiver side will be set during the pairing when the user enters the connection string, that is made on the sender side, on the receiver side.

Not sure why we need to know key uid on the receiver side and set `conf.ReceiverConfig.LoggedInKeyUID` when the user won't be log in ever before the pairing process is done. There is no feature to run pairing/syncing explicitly when the same user is logged in on both devices.

Reply:
@saledjenic it's also possible if there is no info about that account on another device.
you don't have to set `conf.ReceiverConfig.LoggedInKeyUID` in this case, it's optional, it's only needed to be set when user logged in so that we can check if the account we received is different, we don't allow this [case](https://github.com/status-im/status-go/issues/4024) happen ATM.